src/defaults.sh

```diff
@@ -79,7 +79,10 @@ VERBOSITY_TRACE=10
 [ -z "$SQM_DEBUG" ] && SQM_DEBUG=0
 if [ "$SQM_DEBUG" -eq "1" ]
 then
-    SQM_DEBUG_LOG=${SQM_STATE_DIR}/${IFACE}.debug.log
+    SQM_DEBUG_STEM=${SQM_STATE_DIR}/${IFACE}
+    SQM_START_LOG=${SQM_DEBUG_STEM}.start-sqm.log
+    SQM_STOP_LOG=${SQM_DEBUG_STEM}.stop-sqm.log
+    [ -z "SQM_DEBUG_LOG" ] &&  SQM_DEBUG_LOG=${SQM_DEBUG_STEM}.debug.log
     OUTPUT_TARGET=${SQM_DEBUG_LOG}
```

Comment:
These really should be quoted everywhere...

Reply:
The ${SQM_DEBUG_LOG} you mean? I will have a go and test that later.

Reply:
Yeah; just in case someone decides to put a space in their SQM_STATE_DIR path :)

Reply:
Fixed